extensions/interactions/LogicProof/directives/logic_proof_short_response_directive.html

```diff
@@ -4,5 +4,6 @@
 </span>
 <span ng-if="!answer.correct">
   <span translate="I18N_INTERACTIONS_LOGIC_PROOF_INCORRECT_ANSWERS"></span>:
-  <pre><[answer.displayed_proof[1] | truncateAtFirstLine]></pre>
+  <pre><[answer.displayed_proof.join('\n') | truncateAtFirstLine]></pre>
 </span>
+
```

Comment:
remove this line.
@apb7 can you please check how this PR made reached here without failing pre_push linter tests?